lib/ResourceSet.h

```diff
@@ -148,6 +148,26 @@ class ResourceSet
 		return ret;
 	}
 
+	int div(const ResourceSet& income) {
```

Comment:
Method probably needs better name, or at least a comment to explain what it does.

Returns how many items of "this" we can afford with provided income?

Reply:
Method got renamed and got a comment describing what it does.